Library/Formula/mysql.rb

```diff
@@ -60,11 +60,11 @@ def install
       -DCMAKE_FIND_FRAMEWORK=LAST
       -DCMAKE_VERBOSE_MAKEFILE=ON
       -DMYSQL_DATADIR=#{datadir}
-      -DINSTALL_INCLUDEDIR=include/mysql
-      -DINSTALL_MANDIR=share/man
-      -DINSTALL_DOCDIR=share/doc/#{name}
-      -DINSTALL_INFODIR=share/info
-      -DINSTALL_MYSQLSHAREDIR=share/mysql
+      -DINSTALL_INCLUDEDIR=#{include}/mysql
```

Comment:
You may need to revert some of these. I've just noticed the little hidden line above the `args` which says: `# -DINSTALL_* are relative to prefix`. `include`, `man` etc provide full paths rather than relative paths, I believe. Might be the cause of the issue.

I don't have the time to check this out tonight, but I'll put it on the list to poke my head into tomorrow.